wondrous-app/pages/collaboration/[username]/boards.lazy.tsx

```diff
@@ -543,8 +555,21 @@ function BoardsPage() {
     }));
   }
 
-  const handleFilterChange: any = (filtersToApply = { statuses: [], podIds: [], labelId: null, date: null }) => {
-    setFilters(filtersToApply);
+  const handleFilterChange: any = (filtersToApply = { statuses: [], podIds: [], labelId: null, date: null, fromDate: null, toDate: null }) => {
+    setFilters({
+      ...filtersToApply,
+      fromDate: filtersToApply.fromDate ?? filters.fromDate,
+      toDate: filtersToApply.toDate ?? filters.toDate,
+    });
+  };
+
+  const handleActiveViewChange = (newView: ViewType) => {
+    setActiveView(newView);
+
+    if (newView === ViewType.Calendar) {
```

Comment:
this feels weird to trigger query again? maybe we should return refetch from getOrgTaskBoardTasks and use that instead?

Reply:
I don't like it too, but this is a better way, because:
1. we use 2 hooks `useGetTaskRelatedToUser` and `useGetOrgTaskBoardTasks` for organization board. Each hook listens filters, so you don't need to write aditional conditions to refetch specific query. 
2. in case of `refetch` we must modifty `variables` and pass `fromDate` and `toDate` fields, this is a case when you visit grid/list view and it loads first 10 records, but when you switch to the calendar view we should get all records for the specified period (month or week).